packages/nimble-components/src/card/index.ts

```diff
@@ -0,0 +1,24 @@
+import { DesignSystem, Card as FastCard } from '@microsoft/fast-foundation';
```

Comment:
We generally rename the fast component as `FoundationXYZ`, so in this case, you'd do `Card as FoundationCard`.